_layouts/product-get-started.html.slim

```diff
@@ -71,6 +70,10 @@ ol
     h3.divider#ExtraSection #{page._extra_section_title}
   = page._extra_section
 
+h3.divider#bpm_suite_starter_kit JBoss BPM Suite Starter Kit
+.bpm-suite
+  = partial "bpm_suite_starter_kit.adoc"
```

Comment:
This content should live with the BPM Suite product files (/products/bpmsuite). See later for more details...